data/data-pipeline/data_pipeline/etl/sources/cdc_svi_index/etl.py

```diff
@@ -0,0 +1,138 @@
+        )
+
+        self.df = pd.read_csv(
+            filepath_or_buffer=tmp_svi_csv_file_path,
+            dtype={self.CDC_SVI_INDEX_TRACTS_FIPS_CODE: object},
+        )
+
+        # Note: In this dataset all US census tracts are ranked against one another.
+        # Puerto Rico is not included in this dataset
+        self.df.rename(
+            columns={
+                self.CDC_SVI_INDEX_TRACTS_FIPS_CODE: self.GEOID_TRACT_FIELD_NAME,
+                "RPL_THEME1": field_names.CDC_SVI_INDEX_SE_THEME_PERCENTILE_FIELD,
+                "RPL_THEME2": field_names.CDC_SVI_INDEX_HOUSEHOLD_THEME_PERCENTILE_COMPOSITION_FIELD,
+                "RPL_THEME3": field_names.CDC_SVI_INDEX_LANGUAGE_THEME_PERCENTILE_FIELD,
+                "RPL_THEME4": field_names.CDC_SVI_INDEX_HOUSING_TRANSPORTATION_PERCENTILE_FIELD,
+                "RPL_THEMES": field_names.CDC_SVI_INDEX_RPL_THEMES_OVERALL_PERCENTILE_FIELD,
+                "SPL_THEME1": field_names.CDC_SVI_INDEX_SE_THEME_SOS_FIELD,
+                "SPL_THEME2": field_names.CDC_SVI_INDEX_HOUSEHOLD_THEME_SOS_FIELD,
+                "SPL_THEME3": field_names.CDC_SVI_INDEX_LANGUAGE_THEME_SOS_FIELD,
+                "SPL_THEME4": field_names.CDC_SVI_INDEX_HOUSING_TRANSPORTATION_SOS_FIELD,
+                "SPL_THEMES": field_names.CDC_SVI_INDEX_THEMES_OVERALL_SOS_FIELD,
+                "F_TOTAL": field_names.CDC_SVI_INDEX_THEMES_OVERALL_TOTAL_FIELD,
+            },
+            inplace=True,
+            errors="raise",
+        )
+
+        #  At or above 90 for percentile rank
+        self.df[field_names.CDC_SVI_INDEX_THEMES_PRIORITY_COMMUNITY] = (
+            self.df[field_names.CDC_SVI_INDEX_RPL_THEMES_OVERALL_PERCENTILE_FIELD]
+            >= self.CDC_RPL_THEMES_THRSHOLD
+        )
+
+        expected_census_tract_field_length = 11
+        if (
+            not self.df[self.GEOID_TRACT_FIELD_NAME]
+            .apply(lambda x: len(str(x)))
+            .eq(expected_census_tract_field_length)
+            .all()
+        ):
+            raise ValueError(
+                f"GEOID Tract must be length of {expected_census_tract_field_length}"
+            )
+
+    def load(self) -> None:
+        logger.info("Saving CDC SVI Index Data")
+
+        # The F-prefixed variables symbolize a flag (as defined in the documentation)
+        # Tracts in the top 10%, i.e., at the 90th percentile of values, are given a value of 1 to indicate high vulnerability.
+        # Tracts below the 90th percentile are given a value of 0.
+        # For a given theme, the flag value is the number of flags for variables comprising the theme. The overall
+        # flag value is calculated for each tract as the number of all variable flags (F_TOTAL)
+        list_of_flag_columns = [
+            x for x in self.df.columns if str(x).startswith("F_")
+        ]
+
+        self.OUTPUT_PATH.mkdir(parents=True, exist_ok=True)
+
+        self.df[self.COLUMNS_TO_KEEP + list_of_flag_columns].to_csv(
```

Comment:
This outputs 40 columns which is probably too many to be useful -- usually we try to only output the most important and valuable data. Maybe just `CDC_SVI_INDEX_THEMES_PRIORITY_COMMUNITY` and a few others?

Reply:
My intent was to capture covariates that may be missing from other data sources. However, for the purpose of the comparison tool, I can remove those. Moreover, since I have documented the sum of series fields, I will go ahead and remove those as well. 

I think the flag columns, percentile rank column and percentile columns for each theme are valuable.

That reduces the number of columns included.